bruno/collections/Rafiki/Examples/Open Payments Without Quote/Create Incoming Payment.bru

```diff
@@ -5,7 +5,7 @@ meta {
 }
 
 post {
-  url: {{receiverOpenPaymentsHost}}/incoming-payments
+  url: {{receiverOpenPaymentsHost}}/{{receiverTenantId}}/incoming-payments
```

Comment:
You can remove the explicit addition of the `receiverTenantId` (or `senderTenantId`) in other requests now that #3187 & #3152 are merged into the feature branch.

I made a [similar comment](https://github.com/interledger/rafiki/pull/3171#discussion_r1943360155) on #3171 for reference.